ts/components/services/ServicesSectionsList.tsx

```diff
@@ -0,0 +1,161 @@
+  },
+  message: {
+    fontSize: customVariables.fontSizeBase,
+    textAlign: "left"
+  },
+  button: {
+    justifyContent: "center",
+    alignItems: "center",
+    paddingTop: 0,
+    paddingBottom: 0
+  },
+  textButton: {
+    paddingLeft: 0,
+    paddingRight: 0
+  },
+  icon: {
+    color: customVariables.brandPrimary,
+    lineHeight: 24
+  },
+  paddingForAnimation: {
+    height: 55
+  }
+});
+
+class ServicesSectionsList extends React.PureComponent<Props> {
+  private listEmptyComponent() {
+    return (
+      this.props.type === "Local" && (
+        <View style={styles.headerContentWrapper}>
+          <Text style={styles.message}>
+            {I18n.t("services.areasOfInterest.selectMessage")}
+          </Text>
+          <View spacer={true} large={true} />
+          <Button
+            small={true}
+            bordered={true}
+            style={styles.button}
+            block={true}
+            onPress={this.props.onChooserAreasOfInterestPress}
+          >
+            <IconFont name="io-plus" style={styles.icon} />
+            <Text style={styles.textButton}>
+              {I18n.t("services.areasOfInterest.addButton")}
+            </Text>
+          </Button>
+          <View spacer={true} extralarge={true} />
+          <Image source={require("../../../img/services/icon-places.png")} />
+          {this.props.paddingForAnimation && (
+            <View style={styles.paddingForAnimation} />
+          )}
+        </View>
+      )
+    );
+  }
+
+  private renderEditButton = () => {
+    return (
+      this.props.type === "Local" && (
+        <View style={styles.headerContentWrapper}>
+          <Button
+            small={true}
+            bordered={true}
+            style={styles.button}
+            block={true}
+            onPress={this.props.onChooserAreasOfInterestPress}
+            disabled={this.props.isRefreshing}
+          >
+            <Text style={styles.textButton}>
+              {I18n.t("services.areasOfInterest.editButton")}
+            </Text>
+          </Button>
+        </View>
+      )
+    );
+  };
+
+  private renderList = () => {
+    const {
+      animated,
+      profile,
+      sections,
+      isRefreshing,
+      onRefresh,
+      onSelect,
+      readServices
+    } = this.props;
+    return (
+      <ServiceList
+        animated={animated}
+        sections={sections}
+        profile={profile}
+        isRefreshing={isRefreshing}
+        onRefresh={onRefresh}
+        onSelect={onSelect}
+        readServices={readServices}
+        isExperimentalFeaturesEnabled={true}
+        ListEmptyComponent={this.listEmptyComponent()}
+      />
+    );
+  };
+
+  public render() {
+    const isOrganizationsFiscalCodesSelected =
+      this.props.type === "Local" &&
+      this.props.organizationsFiscalCodesSelected.fold(false, _ => _.size > 0);
+
+    return (
+      <View style={styles.contentWrapper}>
+        {isOrganizationsFiscalCodesSelected && this.renderEditButton()}
```

Comment:
```suggestion
        {this.renderEditButton()}
```
if you merge all logics inside _**renderEditButton**_ method

Reply:
updated within https://github.com/teamdigitale/io-app/commit/4a5010d53af5dcbf687bc50cc6ec1cff542f3554